_data/navbars/maintenance.yml

```diff
@@ -11,11 +11,13 @@ section:
     path: /maintenance/openshift-upgrade
   - title: OpenStack
     path: /maintenance/openstack-upgrade
-- title: Deploy image options
+- title: Calico image source options
   path: /maintenance/image-options/
   section:
-  - title: Deploy images by digest
+  - title: Install images by registry digest
     path: /maintenance/image-options/imageset
+  - title: Install images using alternate or modified registry configuration
```

Comment:
"alternate registry or modified registry configuration"?

Or maybe just "Configure use of your image registry" ?

I think the existing title isn't quite right somehow

Reply:
Changed to Configure use of your image registry